doc/aursync.md

```diff
@@ -0,0 +1,29 @@
+# aursync
+
+## Name
+
+aursync - download and build AUR packages automatically
+
+## Synopsis
+
+aursync [+-cdi ARG] [+-knut] [--] ARGS...
+
+## Description
+
+Downloads and builds AUR packages automatically. It asks which repository to use in case of multiple local repositories.
```

Comment:
Defining what a "local repository" is would be good, i.e., mention the
file:/// prefix.